test/functional/index.js

```diff
@@ -1,31 +1,34 @@
+'use strict'; // eslint-disable-line
```

Comment:
Without this, Jenkins throws a fit: http://build-eu-00.elastic.co/job/kibana_core_pr/4323/console